earth_enterprise/src/fusion/config/geconfigureassetroot.cpp

```diff
@@ -182,6 +195,9 @@ int main(int argc, char *argv[]) {
     if (fixmasterhost) {
       FixMasterHost(status);
     }
+    if (listvolumes) {
```

Comment:
Minor: There's a `printf` on line 223 below where I can't add a comment from Git Hub. The `Configured /some/assetroot.` message doesn't make much sense when just listing volumes. I don't think it should be shown when `--listvolumes` is specified.